packages/core/src/tech/HlsJsTech.ts

```diff
@@ -113,7 +111,42 @@ export default class HlsJsTech extends BaseTech {
   }
 
   protected onErrorEvent(event, data) {
-    this.emit(PlayerEvent.ERROR, data);
+    let epasData = {
+      category: data?.type, // optional, eg. NETWORK, DECODER, etc.
+      code: "-1",
+      message: "", // optional
+      data: data, // optional
+    }
```

Comment:
I do not think we should call this anything EPAS related, but rather acknowledge it as a self-defined internal error model to align the different errors from the engines.
It can be as useable for the analytics as it might be for error messages on the implementing site - in theory.

Reply:
Point taken. Changed variable name to `errordata`.